pkgs/development/libraries/libixp-hg/default.nix

```diff
@@ -1,24 +1,23 @@
-{ stdenv, fetchhg, txt2tags }:
+{ stdenv, fetchurl, unzip, txt2tags }:
 
 stdenv.mkDerivation rec {
   rev = "148";
   version = "hg-2012-12-02";
   name = "libixp-${version}";
 
-  src = fetchhg {
-    url = https://code.google.com/p/libixp/;
-    sha256 = "1nbnh2ff18fsrs28mx4bfgncq1d1nw5dd6iwhwvv5x2g9w7q5vvj";
-    inherit rev;
+  src = fetchurl {
+    url = https://storage.googleapis.com/google-code-archive-source/v2/code.google.com/libixp/source-archive.zip;
+    sha256 = "0kcdvdcrkw6q39v563ncis6d7ini64xbgn5fd3b4aa95fp9sj3is";
   };
 
   configurePhase = ''
    sed -i -e "s|^PREFIX.*=.*$|PREFIX = $out|" config.mk
   '';
 
-  buildInputs = [ txt2tags ];
+  buildInputs = [ unzip txt2tags ];
```

Comment:
`unzip` exports a setup hook that implements support for unpacking zip archives; that's why it needs to be added, I think.